python/cudf/cudf/_lib/types.pyx

```diff
@@ -94,6 +96,29 @@ SUPPORTED_NUMPY_TO_LIBCUDF_TYPES = {
     np.dtype("timedelta64[ns]"): TypeId.DURATION_NANOSECONDS,
 }
 
+SUPPORTED_NUMPY_TO_PYLIBCUDF_TYPES = {
```

Comment:
Could we construct this table from the previous one?

Reply:
Hmm maybe I didn't consider that. I'm not sure offhand how we'd go from libcudf types to pylibcudf types but it should be possible by accessing the underlying enum values.

Reply:
Done.